### DISCUSSION TITLE
Why declare the devices if it is a hobby with no profit?

### DISCUSSION THREAD

**Discussion by adisorin** [revoxhere/duino-coin#1820] [General]
Duino-coin is a hobby coin from which nothing is earned, I have been told this dozens of times.
Question: If it's a hobby where you put money in but don't earn anything, why do you have to declare how many devices (miners) you have? I don't see any logic in that.

**Comment by JK-Rolling** [comment]
Indeed you're rightly told it is a hobby coin.
To stay true to **real device** mining Duino-Coin, the owner must make sure no simulation/emulation of devices are mining as this poses unfair advantages to these fake worker in cost.

That's why the verification is in place, making sure people are mining using real devices, where the number of devices can be seen from the photo.

**Reply by adisorin** [reply]
Then why do they have so much value in the wallet, from where, with what devices? No one mined anything with any device!!!

Top 10 Richest:
1 | coinexchange | ᕲ 235437783.9248 | $ 7689.39802299
2 | bscDUCO | ᕲ 85495878.5265 | $ 2792.29539267
3 | AlmarRido | ᕲ 35576240.5690 | $ 1161.92001698
4 | Flowen1985 | ᕲ 30855598.8080 | $ 1007.74385707
5 | viviana80 | ᕲ 20160690.6280 | $ 658.44815591
6 | bimbasimba94 | ᕲ 18317539.1320 | $ 598.25082805
7 | aimbert | ᕲ 14928737.7957 | $ 487.57257641
8 | ctecrypto | ᕲ 14847133.1941 | $ 484.90737012
9 | duinousa | ᕲ 14206639.9181 | $ 463.98885972
10 | QQ44142894 | ᕲ 12721943.3926 | $ 415.49867120

**Reply by JK-Rolling** [reply]
they bought Duco from exchange. not mined. the first 2 are the exchange and reserve.

**Comment by adisorin** [comment]
And the conversation must be closed only by the one who opened it!
